velox/docs/functions/json.rst

```diff
@@ -131,6 +131,11 @@ JSON Functions
 
         SELECT json_format(JSON '{"a": 1, "b": 2}')
 
+.. function:: json_parse(varchar) -> json
+
+    Returns the JSON value deserialized from the input JSON text.
+        SELECT json_parse('{"a": 1, "b": 2}');
```

Comment:
Add empty line before SELECT example and ::

Please, generate the docs locally and verify they look good.